### ISSUE TITLE
Issues with installation regarding Eigen3

### ISSUE THREAD

**Issue by Zorobay** [autodiff/autodiff#107]
Hi!

**Disclaimer: I'm new to C++**

I'm trying to install autodiff, but I'm getting errors related to Eigen3. First of all, after running `cmake ..`
 ,cmake was complaining that it could not locate Eigen3. So I cloned Eigen3's repo and followed it's installation instructions. Now, running `cmake ..` seems to work fine, and I get a message that build files have been written to the `build` folder.

Okay, now running `cmake --build . --target install` yields lots of output and errors saying 

`Building Custom Rule C:/Users/xx/autodiff/examples/forward/CMakeLists.txt
  example-forward-gradient-derivatives-using-eigen.cpp
C:\Users\xx\autodiff\examples\forward\example-forward-gradient-derivatives-using-eigen.cpp(6,10): fatal error C1083: Cannot open include file: 'Eigen/Core': No such file or directory [C:\Users\xx\autodiff\build\examples\forward\example-forward-gradient-derivatives-using-eigen.vcxproj]
`
etc...

I'm working on Windows 10.

So I guess installing Eigen3 was not really successful. What could I have done wrong, and why is Eigen3 not mentioned in the installation instructions?

PS: Thanks everyone for working on this project! Really looking forward to using it :)

**Comment by allanleal** [comment]
I'm going to push to the master branch shortly a fix that will allow you to do:

~~~
cmake .. -DEIGEN3_INCLUDE_DIR=/path/to/downloaded/eigen-3.3.7
~~~

and this will allow you to build the examples and tests.

**Comment by allanleal** [comment]
This has been resolved with pull request #108. I'll close this; let me know if the issue persists.
